webapp/utils/subgraph.ts

```diff
@@ -20,7 +20,7 @@ const subgraphIds = {
   [sepolia.id]: process.env.NEXT_PUBLIC_SUBGRAPH_SEPOLIA_ID,
 }
 
-type GraphResponse<T> = { data: T }
+type GraphResponse<T> = { data: T; errors?: Array<{ message: string }> }
```

Comment:
I think this should be more like this

```suggestion
type GraphResponse<T> = { data: T } | { errors: { message: string }[] }
```